src/code/z_viscvg.c

```diff
@@ -1,70 +1,100 @@
-#include "global.h"
+/**
+ * @file z_viscvg.c
+ * Description: Visualise Coverage (how much of the neighbourhood of a pixel is covered by a primitive) in various ways.
```

Comment:
```suggestion
 * Description: Visualise Coverage (how much of a pixel is covered by a primitive) in various ways.
```
Also idk how much detail we want in these files, as it's not supposed to be a tutorial to N64 graphics, but I also think these are low level and technical enough it's okay to go into details

With that said if you want to explain coverage I think it's worth mentioning its main (only?) use is for antialiasing (for use by the antialiasing and divot filters) (linking to the relevant Prerender docs may be a good idea)
(i.e. if you disable antialiasing coverage is always full everywhere)

Reply:
I had a go, I thought I had reasonable terminology but the manual tripped me up again.